### ISSUE TITLE
1.20.1 - Stuck Spawning 1 type of Flower ``Silent Gems Rose Peridot`` when using ``collective-1.20.1-6.72.jar`` and ``randombonemealflowers-1.20.1-4.3.jar`` 

### ISSUE THREAD

**Issue by P3rf3ctXZer0** [ricksouth/serilum-mc-mods#2018]
## **Information**
Minecraft version: 1.20.1
Modloader: Forge
Forge version: 47.1.3
Environment: Singleplayer / Multiplayer

Mod name: Mod
Mod version: 
``collective-1.20.1-6.72.jar``
``randombonemealflowers-1.20.1-4.3.jar``
``MysticalAgriculture-1.20.1-7.0.5.jar``
``Cucumber-1.20.1-7.0.4.jar``
``silent-lib-1.20.1-8.0.0.jar``
``silents-gems-1.20.1-4.6.0.jar``

## **Description**
So after updating to the latest collective and randombonemealflowers it only spawns one kind flower included those mods because I was using Mystical Fertilizer during test though I don't think it will matter. I do know the bugs don't exist when using ``
randombonemealflowers-1.20.1-4.1.jar`` and ``collective-1.20.1-6.66.jar``

## **Crash report**
There is no crash it just breaks function.

## **Modpack**
https://legacy.curseforge.com/minecraft/modpacks/fiction-friction-pixelmon
1.20.1-rc02-20.0.0 (Just update collective and randombonemeal flowers.)

**Comment by OrganEye** [comment]
Have a similar issue with using this mod in conjunction with Ice And Fire. In my case it's the Fire Lily, but it only seems to happen in the plains biome. At any rate, the "blacklist.txt" doesn't seem to actually do anything. In other biomes, all types of flowers will spawn at all times.

**Comment by P3rf3ctXZer0** [comment]
What version are you on? 1.20.1? This bug does not exist on 1.21.1 Neoforge Latest or 21.1.215

**Comment by OrganEye** [comment]
Yeah, 1.20.1. Unfortunately due to my current modlist, updating to a later version would break everything else so it's probably not worth it for just one mod. On the bright side, I managed to fix the blacklist so it now does its job with every other vanilla and modded plant. For whatever reason it just doesn't want to play nice with the plains biome + Ice and Fire. But at this point I'll probably just work around it.

**Comment by P3rf3ctXZer0** [comment]
Well best of luck to you.
